examples/audio/melgan_spectrogram_inversion.py

```diff
@@ -0,0 +1,574 @@
+            mel = tf.matmul(tf.square(magnitude), self.mel_filterbank)
+            log_mel_spec = tfio.audio.dbscale(mel, top_db=80)
+            return log_mel_spec
+        else:
+            return audio
+
+    def get_config(self):
+        config = super(MelSpec, self).get_config()
+        config.update(
+            {
+                "frame_length": self.frame_length,
+                "frame_step": self.frame_step,
+                "fft_length": self.fft_length,
+                "sampling_rate": self.sampling_rate,
+                "num_mel_channels": self.num_mel_channels,
+                "freq_min": self.freq_min,
+                "freq_max": self.freq_max,
+            }
+        )
+        return config
+
+"""
+The Residual Convolutional Block extensively uses dilations and has a total receptive
+field of 27 timesteps per block. The dilations must grow as a power of the `kernel_size`
+to ensure reduction of hissing noise in the output. The network proposed by the paper is
+as follows:
+
+<p align="center">
+<img src="https://i.imgur.com/gENKH91.png" alt="drawing" height="500"/>
+</p>
+"""
+
+# Creating the Residual Stack Layer
+
+def residual_stack(input, filters):
+    """
+    Convolutional residual stack with weight normalization.
+
+    input: filter: int, determines filter size for the residual stack.
+    output: convolutional stack output.
+    """
+    c1 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=1, padding="same"), data_init=False
+    )(input)
+    lrelu1 = layers.LeakyReLU()(c1)
+    c2 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=1, padding="same"), data_init=False
+    )(lrelu1)
+    add1 = layers.Add()([c2, input])
+
+    lrelu2 = layers.LeakyReLU()(add1)
+    c3 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=3, padding="same"), data_init=False
+    )(lrelu2)
+    lrelu3 = layers.LeakyReLU()(c3)
+    c4 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=1, padding="same"), data_init=False
+    )(lrelu3)
+    add2 = layers.Add()([add1, c4])
+
+    lrelu4 = layers.LeakyReLU()(add2)
+    c5 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=9, padding="same"), data_init=False
+    )(lrelu4)
+    lrelu5 = layers.LeakyReLU()(c5)
+    c6 = addon_layers.WeightNormalization(
+        layers.Conv1D(filters, 3, dilation_rate=1, padding="same"), data_init=False
+    )(lrelu5)
+    add3 = layers.Add()([c6, add2])
+
+    return add3
+
+"""
+Each Convolutional Block uses the dilations offered by the residual stack
+and upsamples the input data by the `upsampling_factor`.
+"""
+
+# Dilated Convolutional Block consisting of the Residual stack
+
+def conv_block(input, conv_dim, upsampling_factor):
+    """
+    Dilated Convolutional Block with weight normalization.
```

Comment:
Move line to the first line in the docstring

Docstrings should look like:


```
"""Description, ending with a period.

Args:
    first_arg: Description of first arg.
    second_arg: Description of second arg.

Returns:
    Output description.
"""

This applies to all docstrings in the example.

Reply:
Changed all doc strings to this format